src/components/banner/template.njk

```diff
@@ -1,7 +1,7 @@
 <div class="hmrc-banner">
   <div class="hmrc-organisation-logo">
   <!--[if gt IE 8]><!-->
-    <svg id="f2a14da0-d308-46dc-ad18-2464344bd5d1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" width="26px" height="26px" viewBox="0 0 134.94 134.94" focusable="false" aria-hidden="true">
+    <svg id="f2a14da0-d308-46dc-ad18-2464344bd5d1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" width="26px" height="26px" viewBox="0 0 134.94 134.94" focusable="false" aria-hidden="true" fill="currentColor">
```

Comment:
The fill attribute is a Presentation attribute (https://www.w3.org/TR/SVG/attindex.html#PresentationAttributes) which means we can set this in the CSS rather than as an attribute (https://www.w3.org/TR/SVG/styling.html#StylingUsingCSS) as it keeps all of these colour properties in one place.